cli/create-tauri-app/src/recipes/vanilla.ts

```diff
@@ -0,0 +1,61 @@
+import { Recipe } from "..";
+import { TauriBuildConfig } from "../types/config";
+import { join } from "path";
+//@ts-ignore
+import scaffe from "scaffe";
+import { shell } from "../shell";
+
+export const vanillajs: Recipe = {
+  descriptiveName: "Vanilla.js",
+  shortName: "vanillajs",
+  configUpdate: (cfg) => ({
+    ...cfg,
+    distDir: `../dist`,
+    devPath: `../dist`,
+    beforeDevCommand: `yarn start`,
+    beforeBuildCommand: `yarn build`,
+  }),
+  extraNpmDevDependencies: [],
+  extraNpmDependencies: [],
+  preInit: async ({ cwd, cfg }) => {
+    const version = await shell("npm", ["view", "tauri", "version"], {
+      stdio: "pipe",
+    });
+    const versionNumber = version.stdout.trim();
+    await run(cfg, cwd, versionNumber);
+  },
+};
+
+export const run = (args: TauriBuildConfig, cwd: string, version: string) =>
+  new Promise((resolve, reject) => {
+    const { appName } = args;
+    const templateDir = join(__dirname, "../templates/vanilla");
```

Comment:
since we're using rollup here. `__dirname` will be `create-tauri-app/dist` so either this will be `join(__dirname, "../src/templates/vanilla")` or `templates` dir will be in `src` directory.
```suggestion
    const templateDir = join(__dirname, "../src/templates/vanilla");
```